bitcoin/src/crypto/schnorr.rs

```diff
@@ -44,6 +44,20 @@ impl fmt::Display for TweakedPublicKey {
 pub type UntweakedKeyPair = KeyPair;
 
 /// Tweaked BIP-340 key pair
+///
+/// # Examples
+/// ```
+/// # #[cfg(feature = "rand-std")] {
+/// # use bitcoin::schnorr::{TweakedKeyPair, TweakedPublicKey};
+/// # use bitcoin::secp256k1::{rand, Secp256k1};
+/// # let secp = Secp256k1::new();
+/// # let keypair = TweakedKeyPair::dangerous_assume_tweaked(KeyPair::new(&secp, &mut rand::thread_rng()));
```

Comment:
This is missing an import statement for `KeyPair` - I'm baffled as to how this got through CI.

Reply:
https://github.com/rust-bitcoin/rust-bitcoin/pull/1610, should have done this yesterday.